modules/core/src/layers/xviz-layer.js

```diff
@@ -331,7 +382,10 @@ export default class XVIZLayer extends CompositeLayer {
               length: data[0].points.length / 3,
               attributes: {
                 getPosition: data[0].points,
-                getColor: data[0].colors
+                getColor: {
```

Comment:
This is fixing the difference between rgb and rgba? We should make sure we have a test for this

Reply:
yep, I will add some tests